src/evo/mnhftx.cpp

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2017-2021 The Dash Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <evo/mnhftx.h>
+#include <llmq/commitment.h>
+#include <llmq/signing.h>
+#include <evo/specialtx.h>
+#include <consensus/validation.h>
+
+#include <chain.h>
+#include <chainparams.h>
+#include <logging.h>
+#include <validation.h>
+
+extern const std::string CBLSIG_REQUESTID_PREFIX = "clsig";
+
+bool MNHFTx::Verify(const CBlockIndex* pQuorumIndex, bool checkSigs) const
```

Comment:
checkSigs is unused here, I think we can just drop it, I think we should always validate sigs here?